### ISSUE TITLE
Stringify throws if the value isn't an Object

### ISSUE THREAD

**Issue by tobiasandersen** [json5/json5#154]
If I do `JSON5.stringify(json)` when `json` doesn't have `Object.prototype`, it will fail. I can work around it by doing something like:

```
if (!(json instanceof Object)) {
  json = { ...json }
}

JSON5.stringify(json)
```

But I feel like this library should be able to stringify JSON structures event if they aren't Objects. If you don't agree, feel free to close this.

**Comment by jordanbtucker** [comment]
JSON5 is able to stringify objects, arrays, strings, numbers, Booleans, and nulls. Anything else, including functions and symbols, should return `undefined`. It should only throw when you have a circular reference or when the `replacer` argument is truthy but neither function nor array.

What are you passing into `stringify` that is causing it to throw, and what is the error message?

**Comment by tobiasandersen** [comment]
Here's an example: https://jsfiddle.net/j1156ewj/1/

**Comment by jordanbtucker** [comment]
I see where the error is now. Thanks.

I might patch this soon if it's an easy fix, but in the meantime, you could use [JSONext](https://github.com/jordanbtucker/jsonext). It supports JSON5 syntax but also supports Unicode keys and includes some ES6 features. It doesn't suffer from this bug as you can see in this updated example: https://jsfiddle.net/j1156ewj/2/

**Comment by tobiasandersen** [comment]
Thanks! I've worked around it already, just thought I'd let you know.

**Comment by jordanbtucker** [comment]
Fixed in 35269daf8c8dd257796c1f1dc768bd25175f2c58.
